backup/backup.go

```diff
@@ -52,11 +52,24 @@ func (b BucketManifestWriter) WriteManifest(ctx context.Context, w io.Writer) er
 
 		dbInfo := b.mc.Database(bkt.ID.String())
 
+		var description *string
+		if bkt.Description != "" {
+			description = &bkt.Description
+		}
+
 		l = append(l, influxdb.BucketMetadataManifest{
-			OrganizationID:         bkt.OrgID,
-			OrganizationName:       org.Name,
-			BucketID:               bkt.ID,
-			BucketName:             bkt.Name,
+			OrganizationID:   bkt.OrgID,
+			OrganizationName: org.Name,
+			BucketID:         bkt.ID,
+			BucketName:       bkt.Name,
+			Description:      description,
+			RetentionRules: []influxdb.RetentionRuleManifest{
+				{
+					Type:                      "expire",
+					EverySeconds:              int64(bkt.RetentionPeriod.Round(time.Second).Seconds()),
```

Comment:
Is there any possibility of losing data or having data mismatches from rounding this / would it matter if there was?

Reply:
The create- and update-bucket APIs also enforce/round to second precision, so this should be safe in practice... that said, we could probably keep this as a raw `time.Duration`. I was originally thinking that the CLI would need to call the create-bucket API, but I'm now working on having restore-bucket do that on the server side. It'd be cleaner overall if this stayed in nanos.

`openapi` PR incoming!

Reply:
influxdata/openapi#131 removes the `RetentionRules` field entirely, because the existing `RetentionPolicies` field contains all the info we need to get retention data as a raw `time.Duration`